setup.py

```diff
@@ -40,7 +41,7 @@ def find_version():
           'argparse_addons>=0.4.0',
           'pyelftools'
       ],
-      python_requires='>=3.6',
+      python_requires='>=3.9',
```

Comment:
Is it necessary to bump this?

Reply:
If the package is no longer tested on 3.6 I wouldn't advertise support for it. Should I add 3.6-3.8 back in the test matrix?

Reply:
I think it is fine without testing on those version for now.

Reply:
ahh ok, was so ling since I used Python so 3.6 is not EOL in my world lol =)